Source/PSMove/Private/FPSMoveWorker.cpp

```diff
@@ -87,132 +185,147 @@ uint32 FPSMoveWorker::Run()
         UE_LOG(LogPSMove, Log, TEXT("PSMove tracker failed to initialize."));
     }
     
-    // TODO: Move this inside the loop to run if PSMoveCount changes.
-    for (int i = 0; i < PSMoveCount; i++)
-    {
-        psmoves.Add(psmove_connect_by_id(i));
-        assert(psmoves[i] != NULL);
-        
-        psmove_enable_orientation(psmoves[i], PSMove_True);
-        assert(psmove_has_orientation(psmoves[i]));
-        
-        WorkerDataFrames[i].IsConnected = true;
-        
-        if (psmove_tracker)
-        {
-            psmove_tracker_set_mirror(psmove_tracker, PSMove_True);
```

Comment:
I noticed you took out the set_mirror line. Any reason for that? I just ran this version of the plugin on my test project and the controller movement was mirrored in the x-axis relative to what I was expecting.